dlt/common/runtime/typing.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any, Callable, Dict, List, Literal, Optional, Sequence, Set, Type, TypedDict, NewType, Union, get_args
+
+
+TExecInfoNames = Literal[
+    "kubernetes",
```

Comment:
look also at this. maybe we can copy code form there to have even more CI envs?
https://www.npmjs.com/package/ci-info

also there's `CI` env flag which says that code runs in CI so maybe we should add "generic_ci"